.github/workflows/build-on-pull-request.yml

```
@@ -13,10 +13,10 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
 
-      - name: Setup JDK 8
+      - name: Setup JDK 17
         uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)